course_catalog/tasks.py

```diff
@@ -34,8 +34,27 @@ def get_mitx_data():
 
 
 @app.task(acks_late=True)
+def get_ocw_courses_with_retry(
```

Comment:
I don't love this solution, but I couldn't find a way to set acks_late conditionally.